packages/next/src/server/app-render.tsx

```diff
@@ -206,6 +206,39 @@ function interopDefault(mod: any) {
   return mod.default || mod
 }
 
+/**
+ * Logs error stack and digest but ignores stack frames irrelevant to the user.
+ */
+function logServerError(isDev: boolean, err: any) {
```

Comment:
Can we grab the original stack frame the same as we do in the dev server here? https://github.com/vercel/next.js/blob/0a8a143ff028732b099cde56f32ce3601d75f963/packages/next/src/server/dev/next-dev-server.ts#L1097